src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -477,6 +477,10 @@
         - name: --enable-custom-ca-trust
           type: bool
           short-summary: Enable Custom CA Trust on agent node pool.
+        - name: --custom-ca-trust-certificates --ca-certs
```

Comment:
```suggestion
        - name: --ca-certs --custom-ca-trust-certificates
```
Maybe change the order and then give it a try?